code/datums/mood_events/_mood_event.dm

```diff
@@ -1,16 +1,29 @@
 /datum/mood_event
+	/// Description of the mood event
 	var/description
+	/// The amount the mood will change
 	var/mood_change = 0
+	/// How long this mood event should last
 	var/timeout = 0
-	var/hidden = FALSE//Not shown on examine
-	var/category //string of what category this mood was added in as
-	var/special_screen_obj //if it isn't null, it will replace or add onto the mood icon with this (same file). see happiness drug for example
-	var/special_screen_replace = TRUE //if false, it will be an overlay instead
+	/// Is this mood event hidden on examine
+	var/hidden = FALSE
+	/// string of what category this mood was added in as
```

Comment:
```suggestion
	/// A category to put multiple mood events. If one of the mood events in the category
	/// is active while another mood event (from the same category) is triggered it will remove
	/// the effects of the current mood event and replace it with the new one
```